src/main/java/com/clickhouse/kafka/connect/sink/db/mapping/Column.java

```diff
@@ -85,6 +85,23 @@ private static Type dispatchPrimitive(String valueType) {
                 break;
             case "Bool":
                 type = Type.BOOLEAN;
+                break;
+            case "Date":
+                type = Type.Date;
+                break;
+            case "Date32":
+                type = Type.Date32;
+                break;
+            case "DateTime":
+                type = Type.DateTime;
+                break;
+            default:
+                if (valueType.startsWith("DateTime64")) {
+                    // Need to understand why DateTime64(3)
```

Comment:
A `precision` parameter is required in the defintion. see https://clickhouse.com/docs/en/sql-reference/data-types/datetime64

Reply:
Since we are not getting it as DataTime it is not relevant. but as Int64